components/CartSidebar.vue

```diff
@@ -0,0 +1,148 @@
+<template lang="pug">
+  div.cart-sidebar-wrapper(:class="openSidebarCart ? 'active' : ''")
+    div.overlay-cart-sidebar(@click="closeSidebarCart()")
+    aside.cart-sidebar
+        div.cart-sidebar-header
+            h4.title.is-3 Cart
+            i.fa.fa-times.fa-lg(@click="closeSidebarCart()")
+        div.cart-sidebar-body
+          div(v-if="total > 0")
+            CartProductListItem(v-for="item in cart",
+                :key="item.name",
```

Comment:
Just wanted to approve it, but noticed one last thing: sometimes the items don't update somehow (tested on my  local API)
Vue warns:
```
50 vue.runtime.esm.js?2b0e:619 [Vue warn]: Duplicate keys detected: 'test'. This may cause an update error.

found in

---> <CartSidebar> at components/CartSidebar.vue
       <Layouts/default.vue> at layouts/default.vue
         <Root>
warn @ vue.runtime.esm.js?2b0e:619
```
I think it should probably use the index for key and not item name which could be non-unique

Reply:
Strange, didn't notice about that sorry! Changed the CartItem for-key in the last commit.

Reply:
Actually it was so in `cart.vue` too. Noticed this accidentally when added all the products at once to the cart